src/cryptonote_core/blockchain.cpp

```diff
@@ -1922,6 +1922,43 @@ void Blockchain::get_output_key_mask_unlocked(const uint64_t& amount, const uint
   unlocked = is_tx_spendtime_unlocked(m_db->get_tx_unlock_time(toi.first));
 }
 //------------------------------------------------------------------
+bool Blockchain::get_output_distribution(uint64_t amount, uint64_t from_height, uint64_t &start_height, std::vector<uint64_t> &distribution, uint64_t &base) const
+{
+  // rct outputs don't exist before v3
+  if (amount == 0)
+  {
+    switch (m_nettype)
+    {
+      case STAGENET: /* start_height = stagenet_hard_forks[2].height; break; */ return false;
```

Comment:
Why is this commented out? The fork height is defined for stagenet as well.

Reply:
There were no forks for it at the time. I'll change.